controllers/users.js

```diff
@@ -1,22 +1,21 @@
-var express = require('express'),
-    router = express.Router(),
-    User = require('../models/user');
+var express = require('express');
+var router = express.Router();
+var User = require('../models/user');
 
-
-// Get a User
+// Get a user by id
 router.get('/:id', function(req, res) {
   User.get(req.params.id).then(function(user) {
-    res.json({user: user});
+    res.status(200).json({user: user});
   }).catch(function(error) {
-    res.status(500).json({error: error}); 
+    res.status(500).json({error: error});
   });
 });
 
-// Create a new User.
+// Create a new user
 router.post('/', function(req, res) {
   if (req.body.hasOwnProperty('email') && req.body.hasOwnProperty('password')) {
```

Comment:
So is this if statement the only way to do this? Would it be possible to make it follow this structure instead:

if (doesn't have correct properties) {
   error;
   return;
}
success code here;

I've always thought that structure is a little cleaner, but obviously this is super nitpicky.